go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/oasisprotocol/curve25519-voi v0.0.0-20221003100820-41fad3beba17
 	github.com/prometheus/client_golang v1.14.0
 	github.com/prometheus/common v0.39.0
-	github.com/pyroscope-io/pyroscope v0.35.1
+	github.com/pyroscope-io/pyroscope v0.36.0
```

Comment:
please remove go.mod / go.sum and make a separate PR for this. this change is big enough on its own. i'd like to separate causes of failures in different PRs.

Reply:
👍

Reply:
👍